src/components/NavBar/NavBar.jsx

```diff
@@ -5,6 +5,25 @@ import { Link, withRouter } from 'react-router-dom';
 import './NavBar.css';
 
 class NavBar extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            menuOpen:false
+        }
```

Comment:
Not a major change, but if I won't tell you to do this, the eslint on my local will make it all red 🙆‍♂️

```suggestion
        super(props);
        this.state = {
            menuOpen:false
        };
```

Apologies, it's my bad that I don't have an eslintrc in this project to enforce lint rules, it's a good idea to add one though, I've opened another issue for that if you want to take that up - #38